app/assets/javascripts/components/containers/mastodon.jsx

```diff
@@ -56,6 +56,7 @@ import uk from 'react-intl/locale-data/uk';
 import zh from 'react-intl/locale-data/zh';
 import bg from 'react-intl/locale-data/bg';
 import { localeData as zh_hk } from '../locales/zh-hk';
+import bg from 'react-intl/locale-data/ar';
```

Comment:
@Marrakchino please take file directly from Master branch and not by forking the repo I forked.
A lot of files and strings changed during these last days :)
So please go master then submit patches :)
Thank you.